arcgis-ios-sdk-samples/Layers/Query a map image sublayer/QueryMapImageSublayer.storyboard

```diff
@@ -24,36 +23,50 @@
+                                <rect key="frame" x="8" y="28" width="359" height="49"/>
                                 <view key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" insetsLayoutMarginsFromSafeArea="NO" id="cgR-8z-ZuK">
-                                    <rect key="frame" x="0.0" y="0.0" width="359" height="46"/>
+                                    <rect key="frame" x="0.0" y="0.0" width="359" height="49"/>
                                     <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                                     <subviews>
-                                        <textField opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="left" contentVerticalAlignment="center" borderStyle="roundedRect" placeholder="Population 2000 &gt;" textAlignment="natural" minimumFontSize="17" translatesAutoresizingMaskIntoConstraints="NO" id="Jkh-zh-w1R">
-                                            <rect key="frame" x="8" y="8" width="293" height="30"/>
-                                            <color key="backgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-                                            <nil key="textColor"/>
-                                            <fontDescription key="fontDescription" type="system" pointSize="14"/>
-                                            <textInputTraits key="textInputTraits" autocorrectionType="no" spellCheckingType="no" keyboardType="numberPad" returnKeyType="done" smartDashesType="no" smartInsertDeleteType="no" smartQuotesType="no"/>
-                                            <connections>
-                                                <outlet property="delegate" destination="rtM-Ml-JFc" id="xVF-g1-7SJ"/>
-                                            </connections>
-                                        </textField>
-                                        <button opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="251" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="9Co-ma-VW1">
-                                            <rect key="frame" x="309" y="7" width="42" height="30"/>
-                                            <state key="normal" title="Query"/>
-                                            <connections>
-                                                <action selector="query:" destination="rtM-Ml-JFc" eventType="touchUpInside" id="T1j-k2-zy8"/>
-                                            </connections>
-                                        </button>
+                                        <stackView opaque="NO" contentMode="scaleToFill" spacing="8" translatesAutoresizingMaskIntoConstraints="NO" id="wbD-0d-BEI">
```

Comment:
The stack view's `alignment` should be `firstBaseline`. You may have tried that and noticed that the spacing with the superview was messed up. That can be fixed by constraining one of the subviews, such as the text field, to the stack view's superview:

`stackView.top = stackView.superview.top + 8` -> `textField.top = stackView.superView.top + 8`
`stackView.superview.bottom = stackView.bottom + 8` -> `stackView.superview.bottom = textField.bottom + 8`